src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -161,17 +162,31 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
   uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
   uint64_t end_ns = thread_state_slice->end_timestamp_ns();
-
-  return absl::StrFormat(
+  std::string tooltip = absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s"
+      "%s",
+      GetThreadStateName(thread_state_slice->thread_state()),
+      GetThreadStateDescription(thread_state_slice->thread_state()));
+  if (thread_state_slice->was_blocked_by_thread() != 0u) {
```

Comment:
As I wrote before, this 0u constant should be defined somewhere else globally. Ideally OrbitBase/ThreadConstants.h if I'm not missing something

Reply:
As discussed in the previous comment we'll discuss it tomorrow.